<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>871</width>
    <height>639</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>274</width>
         <height>580</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QComboBox" name="cb_device">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="freq_layout">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pb_set_cent_freq">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Установить частоту</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gb_server">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Сервер</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1,5">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QTextEdit" name="te_server_info">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_msg">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_send_msg">
             <property name="text">
              <string>Отправить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gb_save_signal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Запись сигнала</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="lb_dropped_buffer">
             <property name="text">
              <string>Потеряно пакетов: 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Путь к файлу</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_signal_filename"/>
           </item>
           <item>
            <widget class="QPushButton" name="bp_set_signal_filename">
             <property name="text">
              <string>Выбрать другой путь</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>947</width>
         <height>580</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="TaPlot" name="widAmlSpecTop" native="true"/>
        </item>
        <item>
         <widget class="TaPlot" name="widSonogramBottom" native="true"/>
        </item>
        <item>
         <widget class="QGroupBox" name="gb_spec_param">
          <property name="title">
           <string>Параметры спектра</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item row="0" column="1" rowspan="2">
            <widget class="QComboBox" name="cb_fftsize">
             <property name="currentText">
              <string>2048</string>
             </property>
             <property name="currentIndex">
              <number>3</number>
             </property>
             <item>
              <property name="text">
               <string>16384</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8192</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4096</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2048</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Размер БПФ</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Размер спектрограммы</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Размер буфера накопления</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="spb_accum_sono">
             <property name="minimum">
              <number>75</number>
             </property>
             <property name="maximum">
              <number>225</number>
             </property>
             <property name="singleStep">
              <number>25</number>
             </property>
             <property name="value">
              <number>75</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cb_accum_spec">
             <property name="currentText">
              <string>96</string>
             </property>
             <property name="currentIndex">
              <number>5</number>
             </property>
             <item>
              <property name="text">
               <string>256</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>224</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>192</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>160</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>128</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>96</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>64</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>32</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="chb_spec_enable">
             <property name="text">
              <string>Отображать спектр</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QCheckBox" name="chb_log_spec">
             <property name="text">
              <string>Логарифмический спектр</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Сервер</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Подключение к серверу</string>
     </property>
     <addaction name="act_admin_connect"/>
     <addaction name="act_client_connect"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Настройки подключения</string>
     </property>
     <addaction name="act_local_host"/>
     <addaction name="act_set_ip"/>
     <addaction name="act_set_port"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="menu_2"/>
    <addaction name="act_disconnect"/>
    <addaction name="separator"/>
    <addaction name="act_get_signals"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_admin_connect">
   <property name="text">
    <string>Администратор</string>
   </property>
   <property name="toolTip">
    <string>Администратор</string>
   </property>
  </action>
  <action name="act_client_connect">
   <property name="text">
    <string>Клиент</string>
   </property>
  </action>
  <action name="act_disconnect">
   <property name="text">
    <string>Отключиться от сервера</string>
   </property>
   <property name="toolTip">
    <string>Отключиться от сервера</string>
   </property>
  </action>
  <action name="act_get_signals">
   <property name="text">
    <string>Получить записи </string>
   </property>
  </action>
  <action name="act_set_ip">
   <property name="text">
    <string>Указать IP адрес сервера</string>
   </property>
  </action>
  <action name="act_set_port">
   <property name="text">
    <string>Указать порт сервера</string>
   </property>
  </action>
  <action name="act_local_host">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Локальная сеть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TaPlot</class>
   <extends>QWidget</extends>
   <header>mTaPlot/mTaPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
